src/components/InputTodo.js

```diff
@@ -18,9 +18,8 @@ class InputTodo extends React.Component {
   handleSubmit = (e) => {
     e.preventDefault();
     const { titleText } = this.state;
-    const { addTodo } = this.props;
     if (titleText.title.trim()) {
-      addTodo.addTodoProps(titleText.title);
+      this.props.addTodoProps(titleText.title);
       this.setState({
         title: '',
       });
```

Comment:
I can't able to create a task in your application when I run it in my local environment. However, I think there is a problem with your `handleSubmit` function.

In your class constructor, you do not have a `titleText` attribute. This is causing an error when you try to distract the state using `titleText` attribute in the `handleSubmit` function.

I would suggest distracting the state by using the `title` attribute